lib/db/mysql/index.js

```diff
@@ -163,7 +163,14 @@ const QUERY_CLIENT_UPDATE = 'UPDATE clients SET ' +
   'trusted=COALESCE(?, trusted), allowedScopes=COALESCE(?, allowedScopes), ' +
   'canGrant=COALESCE(?, canGrant) ' +
   'WHERE id=?';
-const QUERY_CLIENT_DELETE = 'DELETE FROM clients WHERE id=?';
+const QUERY_CLIENT_DELETE = 'DELETE clients, codes, tokens, refreshTokens, clientDevelopers, developers ' +
+  'FROM clients ' +
+  'JOIN codes ON clients.id = codes.clientId ' +
+  'JOIN tokens ON clients.id = tokens.clientId ' +
+  'JOIN refreshTokens ON clients.id = refreshTokens.clientId ' +
+  'JOIN clientDevelopers ON clients.id = clientDevelopers.clientId ' +
+  'JOIN developers ON clients.id = developers.clientId ' +
+  'WHERE clients.id=?';
```

Comment:
Is it worth noting in a comment that this query is on a route that is only available in dev mode?